shared/types/admin/DonationCertificate.ts

```diff
@@ -0,0 +1,8 @@
+export const DONATION_CERTIFICATE_FIRESTORE_PATH = 'donation-certificate';
+
+export type DonationCertificate = {
```

Comment:
We need to make sure that we set the correct firestore rules for this collection, so one can't see the `DonationCertificate` entries of others.